core/src/gl/uniformEntries.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <cstdint>
+#include <memory>
+#include <string>
+
+namespace Tangram {
+
+namespace UniformEntries {
+
+typedef uint16_t EntryId;
+
+struct UniformEntry {
```

Comment:
Right, the only reason I tight them together was for debugging purpose, otherwise it's hard to tell which ID relates to a given uniform name. If this helps making it less verbose I'll update for a global static array.